### PR TITLE
renaming 6 notebooks

### PR DESCRIPTION
**Pull request by adoroshk**
6 notebooks has been renamed and a README file updated per that change

### CODE REVIEW

File: notebook_examples/README.md [hunk 1]
```diff
@@ -293,25 +293,25 @@ This notebook was developed on the conda pack with slug: *generalml\_p37\_cpu\_v
 <sub>Universal Permissive License v 1.0</sup>
 
 ---
-### - PySpark
-#### [`pyspark-data_flow.ipynb`](pyspark-data_flow.ipynb)
+### - Spark NLP within Oracle Cloud Infrastructure Data Flow Studio
+#### [`pyspark-data_flow_studio-spark_nlp.ipynb`](pyspark-data_flow_studio-spark_nlp.ipynb)
```
Comment (jasper-schneider) [inline notebook_examples/README.md:297]: Will this change the `data_flow` tag into a `data_flow_studio` tag? Just double checking if we want that. Would adding `spark_nlp`  be enough to address the jupyter bug with file path lengths?

Comment (jasper-schneider) [reply notebook_examples/README.md:297]: Oh wait, I don't see the actual `pyspark-data_flow_studio-spark_nlp.ipynb` file in this diff. 
Just checking - was it meant to be included in the diff, or did someone change it in a previous PR and forget to update the README, and your README update is just picking that up too?

**Comment by jasper-schneider** [comment]
Looks good to me

File: notebook_examples/README.md (hunk 1 above)
Comment (adoroshk) [reply notebook_examples/README.md:297]: It confused me too at first. The thing is that there are 2 separate notebooks:
a `pyspark-data_flow.ipynb` notebook which is being renamed and moved lower in readme file and `pyspark-data_flow_studio-spark_nlp.ipynb` notebook that stays intact, but in readme file its description moved up and took place of `pyspark-data_flow.ipynb` notebook, that led to change in description, which actually hasn't change, it just 2 notebooks switched places and their descriptions too.

Comment (jasper-schneider) [reply notebook_examples/README.md:297]: Thanks for the clarification, sounds good to me.

**Review by mayoor — APPROVED** [review]

**Pull request merged**